engine/OD/od_angular.h

```diff
@@ -60,24 +62,26 @@ static inline angular_position_t AngularOD_PositionFrom_rad(float rad)
 static inline void AngularOD_PositionToMsg(const angular_position_t *const self, msg_t *const msg)
 {
     msg->header.cmd = ANGULAR_POSITION;
-    memcpy(msg->data, self, sizeof(angular_position_t));
+    memcpy(msg->data, self.private, sizeof(angular_position_t));
```

Comment:
Here you copy the complete struct not only the private value. In this case this is exactly the same thing but it make sense to put `self` instead of `self.private`.

Reply:
got it.